Kentico/kontent-boilerplate-net
# Made compatible with the Delivery SDK v14

### Motivation

Made compatible with: https://github.com/Kentico/kontent-delivery-sdk-net/releases/tag/14.0.0-rc1

### Checklist


### How to test

- test the home controller
- verify that the caching works
  - including the cache invalidation
- verify that the `/sitemap.xml` route works correctly
